saleor/graphql/account/mutations/account.py

```diff
@@ -272,9 +272,24 @@ def perform_mutation(  # type: ignore[override]
             )
         channel_slug = clean_channel(channel, error_class=AccountErrorCode).slug
         manager = get_plugin_manager_promise(info.context).get()
+
+        # Notifications will be deprecated in the future
         notifications.send_account_delete_confirmation_notification(
             redirect_url, user, manager, channel_slug=channel_slug
         )
+
+        token = account_delete_token_generator.make_token(user)
```

Comment:
Here we generate a second token for the user account delete confirmation link. Should we use the same link in both ways to send this notification?